client/src/components/DatePicker/DatePicker.css

```diff
@@ -0,0 +1,29 @@
+@font-face {
+  font-family: Raleway;
+  src: url("https://fonts.googleapis.com/css?family=Raleway&display=swap");
+}
+
+label {
+  display: block;
+  font-family: Raleway;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 18px;
+  line-height: 21px;
+}
+
+input,
+label {
+  margin: 0.4rem 0;
+}
+
+#requestedDate {
```

Comment:
Can you use a class here? Also, classes and IDs should be all lowercase and separated by dashes (i.e. `request-date`, rather than `requestDate`)

Reply:
Dome